data/global/excel/misc.txt

```diff
@@ -146,8 +146,8 @@ Diablo's Horn	0	100	0	0	0			1	0	0	1	80		dhn	fng	dhn	16	1	1	0	0	0	flpfang	invfang
 Baal's Eye	0	100	0	0	0			1	0	0	1	80		bey	eyz	bey	16	1	1	0	0	0	flpeye	inveye		0	xxx			0	ques		item_monsterbone	12	item_monsterbone	0	0	5	0	0	0	0	0	0	0			0	-1	-1																0	0	0	non	0					255					255					255					255					255					255					255					255					255					255					255					255					255					255					255					255					255	0	0	0	xxx	xxx						
 Mephisto's Brain	0	100	0	0	0			1	0	0	1	80		mbr	brz	mbr	16	1	1	0	0	0	flpbrnz	invbrnz		0	xxx			0	ques		item_monsterbone	12	item_monsterbone	0	0	5	0	0	0	0	0	0	0			0	-1	-1																0	0	0	non	0					255					255					255					255					255					255					255					255					255					255					255					255					255					255					255					255					255	0	0	0	xxx	xxx						
 Token of Absolution	0	100	0	0	0			999	0	0	1	99999		toa	toa	toa	16	1	1	0	0	0	flprun	invtoa		0	xxx			1	ques		item_amulet	14	item_book	0	0	5	0	0	0	0	0	0	0			0	-1	13											1	UseTokenOfAbsolution	UseTokenOfAbsolutionController		8	0	0	0	non	0					255					255					255					255					255					255					255					255					255					255					255					255					255					255					255					255					255	0	0	0	xxx	xxx						
-Twisted Essence of Suffering	0	100	0	0	0			1	0	0	1	10000		tes	fng	tes	16	1	1	0	0	0	flpgsb	invtes		0	xxx			0	ques		item_rare	12	item_rare	0	0	5	0	0	0	0	0	0	0			0	-1	-1																0	0	0	non	0					255					255					255					255					255					255					255					255					255					255					255					255					255					255					255					255					255	0	0	0	xxx	xxx						
-Charged Essense of Hatred	0	100	0	0	0			1	0	0	1	10000		ceh	eyz	ceh	16	1	1	0	0	0	flpgsy	invceh		0	xxx			0	ques		item_rare	12	item_rare	0	0	5	0	0	0	0	0	0	0			0	-1	-1																0	0	0	non	0					255					255					255					255					255					255					255					255					255					255					255					255					255					255					255					255					255	0	0	0	xxx	xxx						
+Essence of Socketing	0	100	0	0	0			1	1	0	1	10000		tes	fng	tes	16	1	1	0	0	0	flpgsb	invtes		0	xxx			0	ques		item_rare	12	item_rare	0	0	5	0	0	0	0	0	0	0			0	-1	-1																0	0	0	non	0					255					255					255					255					255					255					255					255					255					255					255					255					255					255					255					255					255	0	0	0	xxx	xxx						
+Essence of Conversion	0	100	0	0	0			1	1	0	1	10000		ceh	eyz	ceh	16	1	1	0	0	0	flpgsy	invceh		0	xxx			0	ques		item_rare	12	item_rare	0	0	5	0	0	0	0	0	0	0			0	-1	-1																0	0	0	non	0					255					255					255					255					255					255					255					255					255					255					255					255					255					255					255					255					255	0	0	0	xxx	xxx						

```

Comment:
Instead of editing a vanilla item, lets clone it and have our own and then update the drop tables to only drop the new.
This is safer and prevents us from running into engine issues and also allows others who have custom recipes to continue to work with them.